### PR TITLE
Fix the dependencies

### PR DESCRIPTION
**Pull request by faceair**
Previous discussions https://github.com/jaegertracing/jaeger-operator/pull/1257

### CODE REVIEW

File: .ci/vgot.sh [hunk 1]
```diff
@@ -0,0 +1,15 @@
+#!/bin/sh
+if [ $# = 0 ]; then
+    usage: vgot cmdpackage[@version]... >&2
+    exit 2
+fi
+d=`mktemp -d`
+cd "$d"
+echo 'module temp' > go.mod
```
Comment (jpkrohling) [inline .ci/vgot.sh:8]: what's the advantage of this vs. `go mod init temp`?

Comment (faceair) [reply .ci/vgot.sh:8]: There is no real difference, if you prefer `go mod init temp` I can change it to this

Comment (jpkrohling) [reply .ci/vgot.sh:8]: I think I would prefer it, yes.

Comment (faceair) [reply .ci/vgot.sh:8]: It should be faster.

**Review by jpkrohling — APPROVED** [review]

**Pull request merged**